src/main/java/frc/robot/autos/MiddleScoreEngage.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.autos;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.lib.util.FieldConstants;
+import frc.robot.commands.arm.DockArm;
+import frc.robot.commands.drive.MoveToPos;
+import frc.robot.commands.drive.MoveToScore;
+import frc.robot.commands.wrist.WristIntakeRelease;
+import frc.robot.subsystems.Arm;
+import frc.robot.subsystems.Swerve;
+import frc.robot.subsystems.WristIntake;
+
+/**
+ * Score one game piece then engage from community side.
+ */
+public class MiddleScoreEngage extends SequentialCommandGroup {
```

Comment:
Was this actually tested? If not may want to move it to its own PR or at least remove it from the drop down so it's not accidentally run before testing

Reply:
Was not tested. It should be moved to its own branch

Reply:
Aidan did this.